app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> bool:
+    if not command.startswith("cp") or len(command.split(" ")) != 3:
```

Comment:
What if command will start with "cpu..."?

Reply:
But give me example where cpu command can be given in function wich works with file